python_requests/example.py

```diff
@@ -0,0 +1,81 @@
+"""
+Usage
+
+python example.py <hotel_id> <partner_id> <api_key>
+"""
+import argparse
+import base64
+
+import requests
+
+
+def get_auth_token(partner_name, api_key):
+    """Returns the API authorization token. Use this for subsequent calls.
+ 
+    When the token expires, you can get another one using logic similar to what this function
+    illustrates
+
+    :param str partner_name: the partner's name for use in the API
+    :param str api_key: the partner's API key, for use in the API
+
+    :rtype: str
+    :return: A JWT token
+    """
+    authorization_raw_key = '{}:{}'.format(partner_name, api_key)
+
+    auth_response = requests.post(
+        'https://analytics.trustyou.com/connect/api/v1.0/auth',
```

Comment:
Noob Friendly comment #1, can you offload the URLs into `url` variables defined before the `requests.post` method?

The signature of the function call will look like: `requests.post(url, ...)`.